src/test/java/GildedRoseTest.java

```diff
@@ -1,10 +1,39 @@
 import static org.junit.jupiter.api.Assertions.*;
 import org.junit.jupiter.api.Test;
 
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.HashMap;
 
 class GildedRoseTest {
     @Test
     void gildedRoseUpdatesEachItemWithTheAppropriateUpdater() {
+        String brieItemName = "Brie";
+        String baguetteItemName = "Baguette";
+        Item brieItem = new Item(brieItemName, 0, 0);
+        Item baguetteItem = new Item(baguetteItemName, 0, 0);
+        Item[] items = new Item[] { brieItem, baguetteItem };
+        HashMap<String, Updateable> updaters = new HashMap<String, Updateable>();
+        updaters.put(brieItemName, new BrieUpdater());
+        updaters.put(baguetteItemName, new BreadUpdater());
+        GildedRose app = new GildedRose(items, updaters);
+        app.updateQuality();
```

Comment:
**suggestion (non-blocking):** this is quite a big test, which is not a problem in itself and I don't think needs any changes here, but it might be nice in future projects to use the Arrange/Act/Assert pattern as a way to help future maintainers 😄

Reply:
Sounds good